### ISSUE TITLE
Camera Not Working with when some other app using camera

### ISSUE THREAD

**Issue by ravigangwar1991** [Awalz/SwiftyCam#203]
I'm Facing a issue when some other app used my camera then camera session stop working

**Comment by ShaikBaji1506** [comment]
yes because already other app is being used by the camera so make sure your session starts running and  all those cases you were handled or not
